### DISCUSSION TITLE
Hardware shopping for IBEX - experiences needed!

### DISCUSSION THREAD

**Discussion by eliisakek** [IBEXImagingCommunity/ibex_imaging_knowledge_base#234] [Q&A]
HI!

I am setting up IBEX to my lab and first I must buy a suitable imaging system that could enable the automated IBEX with Fluigent Aria system. I must go through pan-EU tendering and I am a bit worried that I might end up getting several offers. Anyone in the community doing automated IBEX with some other imaging system than Leica Thunder? I would gladly hear your experiences and reduce/increase my anxiety related to this massive shopping task...

**Comment by radtkea** [comment]
Hi Eliisakek,

I don't know anyone doing automated IBEX with a system other than the Leica THUNDER. Colleagues in the field are adapting our Nature Protocols that utilizes the THUNDER. Members of the IBEX community are using other systems (**manually**): a slide scanner (Zeiss), a high-end Zeiss confocal instrument for spectral imaging, and a high-content imager from Molecular Devices. I also used the Cell DIVE imager and IBEX dye inactivation protocol for this [publication](https://www.cell.com/cancer-cell/abstract/S1535-6108(24)00045-X). 

Sorry you are anxious. You could always reach out to Fluigent. 

One last thing, the THUNDER was recently announced to support [spinning disk](https://www.labmate-online.com/news/news-and-views/5/crestoptics/strategic-collaboration-extends-spinning-disk-microscopy-capabilities/63444) capabilities!

**Comment by wolheu** [comment]
Hi Eliisakek, 

We have set up and tested the IBEX Protokoll with the Zeiss AxioObserver7 with Zen Blue and the Aria. The software Zen Blue and the Aria communicates via TCP relyably (basically plug and play) The Sample Finder does perfectly fit in the concept - automatic overview image with contrast, sample holder and focus detection  - perfect for defining tile reagions of multiple tissue sections.  All Zen Blue driven systems are easyly connected (e. g. LSMs, SIM Family). 
I'm happy to share my experience from Basel, Bern and Feldbach. 

Best regards, 
Wolf
